### PR TITLE
Specify year{s} inline

### PR DESCRIPTION
**Pull request by umer936**
Potentially better solution than including a dependency for just 1 expression. Fixes https://github.com/jsrn/howoldisit/issues/72.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ class App extends Component {
     for (let i = 0; i < this.state.technologies.length; i++) {
       rows.push(
         <p key={this.state.technologies[i].name} style={this.rowStyle(this.state.technologies[i].name)}>
-          <strong>{this.state.technologies[i].name}</strong> has been out for <strong>{this.daysSince(this.state.technologies[i].released)} years</strong>
+          <strong>{this.state.technologies[i].name}</strong> has been out for <strong>{this.daysSince(this.state.technologies[i].released)} year{(this.daysSince(this.state.technologies[i].released) !== 1) ? "s":""}</strong>
         </p>
```
Comment (disissid) [inline src/App.js]: I think it would be applicable to 0 too.

Comment (umer936) [reply src/App.js]: I thought about/tried that and I don't think so. 

"This project has been out for 0 year" vs  "This project has been out for 0 years" 
The first one is not grammatically correct.

Comment (umer936) [reply src/App.js]: An alternative could be "This project has been out for <1 year", but if we're sticking with "0", it should be "years".

Comment (disissid) [reply src/App.js]: I like "<1 year" approach more

Comment (Esteban-Rocha) [reply src/App.js]: I agree with @umer936 this looks better than including an additional dependency, @disissid there is also #75 so, with that being said let me know which one to merge and also I believe too that this should be "<1 year" or just plain "less than 1 year"

Comment (jsrn) [reply src/App.js]: I like the format "less than a year" since it's quite conversational and suits the overall style of the page.

Comment (disissid) [reply src/App.js]: @Esteban-Rocha I've updated #75 for 0 year case

**Comment by jsrn** [comment]
Thank you for the contribution, but I've merged another user's PR, since the initially expressed interest in tackling the problem.

Sorry for any inconvenience!

**Pull request closed without merging**